1_html-css/styles.css

```diff
@@ -0,0 +1,411 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background-color: #f1f1f1;
+  color: #0f141e;
+  font-size: 16px;
+  font-weight: normal;
+  font-family: "Roboto", sans-serif;
+}
+
+h2 {
+  text-align: center;
+}
+
+.about {
+  color: #5e5e5e;
+  margin: 5px;
+}
+
+.about a {
+  border-bottom: 2px dotted lightcoral;
+  line-height: 36px;
+}
+
+.about a:hover {
+  text-decoration: none;
+  border-bottom: 2px solid blue;
+  line-height: 36px;
+}
+
+.info {
+  display: flex;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+@media (max-width: 700px) {
+  .info {
+    display: block;
+    text-align: center;
+  }
+}
+
+a {
+  color: black;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+a:active {
+  color: blue;
+}
+a:visited {
+  color: #4b4d4c;
+}
+.avatar {
+  width: 200px;
+  border-radius: 50%;
+  box-shadow: 4px 4px 14px rgba(0, 0, 0, 0.25);
+}
+
+@media (max-width: 700px) {
+  .avatar {
+    width: 40%;
+    border-radius: 50%;
+    box-shadow: 4px 4px 14px rgba(0, 0, 0, 0.25);
+  }
+}
+
+.info p {
+  color: #89898f;
+  margin: 0;
+  font-style: italic;
+}
+
+@media (max-width: 700px) {
+  .info p {
+    color: #89898f;
+    margin: 20px;
+    font-style: italic;
+    font-weight: bold;
+  }
+}
+.card {
+  padding: 15px 15px;
+}
+
+.card h2 {
+  margin-bottom: 10px;
+}
+
+.list {
+  padding: 0 20px;
+}
+
+.list li {
+  color: #5e5e5e;
+}
+
+.table {
+  padding: 20px;
+  width: 100%;
+  background-color: rgb(227, 227, 227);
+}
+
+.table td {
+  padding-bottom: 10px;
+}
+
+.table th {
+  text-align: left;
+  font-size: 18px;
+}
+
+.contact h2 {
+  text-align: center;
+}
+
+.form-group label {
+  display: block;
+  font-size: 14px;
+  color: #86898f;
+}
+
+.info-text {
+  margin-left: 40px;
+  max-width: 600px;
+  line-height: 30px;
+}
+
+.describer {
+  max-width: 300px;
+  margin: 0 auto 2rem auto;
+  border-bottom: 0.2rem solid #cbcbcd;
+  font-size: 20px;
+}
+
+.info-texth4 {
```

Comment:
You have a few unused CSS classes. 
Not a huge problem when it is a small project, but remember to try to leave your code clean and tidy as it helps a lot on larger projects in the future :)